solr/core/src/test/org/apache/solr/handler/admin/ZookeeperStatusHandlerTest.java

```diff
@@ -74,6 +78,39 @@ public void tearDown() throws Exception {
     super.tearDown();
   }
 
+  @Test
+  public void testZkread() throws Exception {
```

Comment:
This test doesn't belong in `ZookeeperStatusHandlerTest`

Reply:
That's true. ZKStatusHandler is the handler that pulls 4LW status from ZK TCP API, which is a completely different handler :)